compat/vcbuild/vcpkg_defines.bat

```diff
@@ -0,0 +1,41 @@
+@ECHO OFF
+REM ================================================================
+REM This script builds a set of MAKEFILE definitions to allow a
+REM normal "make MSVC=1 vcpkg" to find the vcpkg-version of the

```

Comment:
This should just be "make MSVC=1".  I wrote this comment when I was trying to keep both types of targets.